projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -277,6 +278,10 @@ public static function enqueue_scripts() {
 						array( 'blocked_logins' => (int) get_site_option( 'jetpack_protect_blocked_attempts', 0 ) )
 					),
 				),
+				'videopress'             => array(
+					'featuredStats' => self::get_videopress_stats(),
+					'videoCount'    => array_sum( (array) wp_count_attachments( 'video' ) ),
```

Comment:
we could include that code in `self::get_videopress_stats()` function?
I think it's a nice practice not to perform any transformations in the array definition itself.

Reply:
That's a good note, I'll get that moved